doc/source/user_guide/groupby.rst

```diff
@@ -1574,8 +1578,8 @@ Plotting
 ~~~~~~~~
 
 Groupby also works with some plotting methods.  For example, suppose we
-suspect that some features in a DataFrame may differ by group, in this case,
-the values in column 1 where the group is "B" are 3 higher on average.
+suspect that some features in a DataFrame may differ by group. In this case,
+in group "B", the values in column 1 are 3 times higher on average.
```

Comment:
Agreed on splitting this into sentences to improve the flow. However, with the split it is now not clear to me that we are "suspecting" the values in column 1 are 3 times higher on average. The current version here sounds like a statement of fact. 

Maybe something like

> In this case, suppose we suspect that the value in column 1 are 3 times higher on average in group "B".

Reply:
I'll write the way you suggest.  Is it "value" or "values"?

Reply:
values - thanks!